bluemira/geometry/tools.py

```diff
@@ -210,8 +213,42 @@ def make_polygon(
     return BluemiraWire(cadapi.make_polygon(points, closed), label=label)
 
 
-@log_geometry_on_failure
 def make_bspline(
+    poles, mults, knots, periodic, degree, weights, check_rational, label: str = ""
+):
+    """
+    Builds a B-Spline by a lists of Poles, Mults, Knots
+
+    Parameters
+    ----------
+    poles: Union[list, np.ndarray]
+        list of poles.
+    mults: Union[list, np.ndarray]
+        list of integers for the multiplicity
+    knots: Union[list, np.ndarray]
+        list of knots
+    periodic: bool
```

Comment:
Argument needs a description. The one used in the FreeCAD API seems to be correct (sans the question). 
[Periodic B-splines](https://en.wikiversity.org/wiki/CAGD/B-splines#Periodic_B-splines)